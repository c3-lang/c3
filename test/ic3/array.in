(U8) {0, 1}
a = (U8) {0, 1}
quote a[0]
quote a[1]
a[0]
a[1]
(U8) {{0, 1},
      {2, 3}}
b = (U8) {{0, 1},
          {2, 3}}
quote b[0][0]
quote b[0][1]
quote b[1][0]
quote b[1][1]
b[0][0]
b[0][1]
b[1][0]
b[1][1]
c = (U8) {{{0, 1},
           {2, 3}},
          {{4, 5},
           {6, 7}}}
quote c[0][0][0]
quote c[0][0][1]
quote c[0][1][0]
quote c[0][1][1]
quote c[1][0][0]
quote c[1][0][1]
quote c[1][1][0]
quote c[1][1][1]
c[0][0][0]
c[0][0][1]
c[0][1][0]
c[0][1][1]
c[1][0][0]
c[1][0][1]
c[1][1][0]
c[1][1][1]
d = (U8) {{{{0, 1},
            {2, 3}},
           {{4, 5},
            {6, 7}}},
          {{{8, 9},
            {10, 11}},
           {{12, 13},
            {14, 15}}}}
quote d[0][0][0][0]
quote d[0][0][0][1]
quote d[0][0][1][0]
quote d[0][0][1][1]
quote d[0][1][0][0]
quote d[0][1][0][1]
quote d[0][1][1][0]
quote d[0][1][1][1]
quote d[1][0][0][0]
quote d[1][0][0][1]
quote d[1][0][1][0]
quote d[1][0][1][1]
quote d[1][1][0][0]
quote d[1][1][0][1]
quote d[1][1][1][0]
quote d[1][1][1][1]
d[0][0][0][0]
d[0][0][0][1]
d[0][0][1][0]
d[0][0][1][1]
d[0][1][0][0]
d[0][1][0][1]
d[0][1][1][0]
d[0][1][1][1]
d[1][0][0][0]
d[1][0][0][1]
d[1][0][1][0]
d[1][0][1][1]
d[1][1][0][0]
d[1][1][0][1]
d[1][1][1][0]
d[1][1][1][1]
