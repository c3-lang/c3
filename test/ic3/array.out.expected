(u8) [0, 1]
(u8) [[0, 1], [2, 3]]
